libs/ledgerjs/packages/cryptoassets/src/crypto-assets-importer/fetch/index.ts

```diff
@@ -1,13 +1,87 @@
+import { getEnv } from "@ledgerhq/live-env";
+import { CryptoCurrencyId } from "@ledgerhq/types-cryptoassets";
 import axios, { AxiosError } from "axios";
 
-const CRYPTO_ASSETS_BASE_URL = "https://cdn.live.ledger.com/cryptoassets";
-
-export const fetchTokens = async <T>(filename: string): Promise<[T, string | undefined]> => {
+export const fetchTokensFromCDN = async <T>(filename: string): Promise<[T, string | undefined]> => {
```

Comment:
As we are switching from CDN to CAL service, should we remove this DYNAMIC_CAL_BASE_URL ?

Reply:
`fetchTokensFromCDN` is still used in some cases like EIP712 and exchange tokens. Those files are going to be removed soon and we'll be able to remove the helper then 👍